### PR TITLE
Allow any order of contents inside XML nodes

### PR DESCRIPTION
**Pull request by moodmosaic**
Addresses #19.

### CODE REVIEW

File: Src/Qaiain.CommandLine/Program.fs [hunk 1]
```diff
@@ -37,21 +37,23 @@ module Mail =
         let ns = XmlNamespaceManager(xml.NameTable)
         ns.AddNamespace("e", "urn:grean:schemas:email:2014")
 
-        let select path = xml.DocumentElement.SelectSingleNode(path, ns).InnerText
-        let selectAll path = xml.DocumentElement.SelectNodes(path, ns)
+        let document = xml.DocumentElement
+
+        let select (x : XmlNode) path = x.SelectSingleNode(path, ns).InnerText
```
Comment (ploeh) [inline Src/Qaiain.CommandLine/Program.fs]: If you switch the arguments around, you'd be able to write

``` F#
node |> select "e:smtp-address"
```

Personally, I think this is easier to read, but it's subjective... What do you think?

Comment (moodmosaic) [inline Src/Qaiain.CommandLine/Program.fs]: I think I like it ;)

Comment (moodmosaic) [inline Src/Qaiain.CommandLine/Program.fs]: Now in bbb0d3e.

File: Src/Qaiain.CommandLine/Program.fs [hunk 2]
```diff
@@ -37,21 +37,23 @@ module Mail =
...
+        let document = xml.DocumentElement
+
+        let select path (x : XmlNode) = x.SelectSingleNode(path, ns).InnerText
+        let selectAll path = document.SelectNodes(path, ns)
 
         try
             {
-                From = { SmtpAddress = select "e:from/e:smtp-address"
-                         DisplayName = select "e:from/e:display-name" }
+                From = { SmtpAddress = document |> select "e:from/e:smtp-address"
```
Comment (moodmosaic) [inline Src/Qaiain.CommandLine/Program.fs:47]: Pretty cool syntax :)

**Pull request merged**